common_helper_extraction/jffs.py

```diff
@@ -22,10 +22,8 @@
 
 def extract_jffs(input_data: bytes) -> list:
     jffs_regex = b'(\x85\x19)|(\x19\x85)'
-    fs_sections = list()
     offset, last_node = get_index(input_data, jffs_regex)
     if (offset, last_node) == (None, None):
-        return fs_sections
-    last_node += get_data_size(input_data[last_node + offset:], 4, 'I', )
-    fs_sections.extend([offset, input_data[offset:last_node]])
-    return fs_sections
+        return ()
```

Comment:
the return value should be a list ⇨ `return []` or `return list()`

Reply:
Changed to `[]`.
Got confused with the new return structure `list(list(tuple()))`